build/guide/intro.md

```diff
@@ -0,0 +1,172 @@
+---
+title: 1. Introduction
```

Comment:
Again, super picky point - I wouldn't have the numbers appear in the title on the page. Our get started guides don't do this either